pkg/cli/cmd/resourceprovider/common/objectformat.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import "github.com/radius-project/radius/pkg/cli/output"
+
+// GetResourceProviderTableFormat returns the fields to output from a resource provider object.
+func GetResourceProviderTableFormat() output.FormatterOptions {
+	return output.FormatterOptions{
+		Columns: []output.Column{
+			{
+				Heading:  "RESOURCE",
+				JSONPath: "{ .Name }",
+			},
+			{
+				Heading:  "TYPE",
+				JSONPath: "{ .Type }",
+			},
+			{
+				Heading:  "STATE",
+				JSONPath: "{ .Properties.ProvisioningState }",
+			},
+		},
+	}
```

Comment:
Should any one of these be `Resource Provider` or just `Provider`?

Reply:
Should we add unit test for this?

Reply:
> Should any one of these be Resource Provider or just Provider?


This is consistent with other formats. We tend to use `RESOURCE` for resource names rather than special casing everything. 

> Should we add unit test for this?

I really hope we don't need to. This function just returns hardcoded data. If Go had better support for static readonly data, I wouldn't use a function at all.